CONTRIBUTING.MD

```diff
@@ -0,0 +1,76 @@
+# Contributing to [project-title]
```

Comment:
the contributing guide seems for Microsoft owned templates, what about templates contributed by external contributor

Reply:
Good question. Since we're not accepting external contributors for the first iterations, for the gallery, we will define those guidelines in future iterations